docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -155,6 +155,24 @@ The directory to write `yb-tserver` log files.
 
 Default: Same as [`--fs_data_dirs`](#fs-data-dirs)
 
+##### --log_min_seconds_to_retain
+
+The minimum number of seconds for which to keep log segments to keep at all times, regardless of what is required for durability. Logs may be retained for a longer amount of time if they are necessary for correct restart. This should be set long enough such that a tablet server which has temporarily failed can be restarted within the given time period. If a server is down for longer than this amount of time, it is possible that its tablets will be re-replicated on other machines.
```

Comment:
The latter part, of side effect if the node is down for longer, is not actually correct. There's 2 flags, which are defaulted to the same value of 15m, which maybe cause some confusion.

`follower_unavailable_considered_failed_sec` -- This flag dictates that if a TS is down for longer than this amount of seconds, then we will remove it from raft groups and re-replicate data somewhere else
`log_min_seconds_to_retain` -- This flag dictates that our GC policy will not remove wal files which contain records up to this time window. If a leader has GCed a WAL file that contains records needed for an active follower, then what will actually happen is that that follower will be forced to go into what we call `Remote Bootstrap`, where it will get all the files from the leader, in order to catch up.

However, in practice, if these flags have the same value, then once that threshold is reached, the unavailability flag kicks in and removes the peer, so the GC aspect of the second flag is irrelevant.

Reply:
The content I used was directly from the online help — want me to add your revisions to a PR to correct that too?